src/garage/tf/models/gaussian_gru_model2.py

```diff
@@ -0,0 +1,246 @@
+"""Gaussian GRU Model.
+
+A model represented by a Gaussian distribution
+which is parameterized by a Gated Recurrent Unit (GRU).
+"""
+import numpy as np
+import tensorflow as tf
+import tensorflow_probability as tfp
+
+from garage.tf.models import Model
+from garage.tf.models.gru import gru
+from garage.tf.models.parameter import recurrent_parameter
+
+
+class GaussianGRUModel2(Model):
+    """Gaussian GRU Model.
+
+    A model represented by a Gaussian distribution
+    which is parameterized by a Gated Recurrent Unit (GRU).
+
+    Args:
+        output_dim (int): Output dimension of the model.
+        hidden_dim (int): Hidden dimension for GRU cell for mean.
+        name (str): Model name, also the variable scope.
+        hidden_nonlinearity (callable): Activation function for intermediate
+            dense layer(s). It should return a tf.Tensor. Set it to
+            None to maintain a linear activation.
+        hidden_w_init (callable): Initializer function for the weight
+            of intermediate dense layer(s). The function should return a
+            tf.Tensor.
+        hidden_b_init (callable): Initializer function for the bias
+            of intermediate dense layer(s). The function should return a
+            tf.Tensor.
+        recurrent_nonlinearity (callable): Activation function for recurrent
+            layers. It should return a tf.Tensor. Set it to None to
+            maintain a linear activation.
+        recurrent_w_init (callable): Initializer function for the weight
+            of recurrent layer(s). The function should return a
+            tf.Tensor.
+        output_nonlinearity (callable): Activation function for output dense
+            layer. It should return a tf.Tensor. Set it to None to
+            maintain a linear activation.
+        output_w_init (callable): Initializer function for the weight
+            of output dense layer(s). The function should return a
+            tf.Tensor.
+        output_b_init (callable): Initializer function for the bias
+            of output dense layer(s). The function should return a
+            tf.Tensor.
+        hidden_state_init (callable): Initializer function for the
+            initial hidden state. The functino should return a tf.Tensor.
+        hidden_state_init_trainable (bool): Bool for whether the initial
+            hidden state is trainable.
+        learn_std (bool): Is std trainable.
+        init_std (float): Initial value for std.
+        std_share_network (bool): Boolean for whether mean and std share
+            the same network.
+        layer_normalization (bool): Bool for using layer normalization or not.
+
+    """
+
+    # pylint: disable=assignment-from-no-return
```

Comment:
please include a link to the known bug which requires this disable

Reply:
After installing latest pylint the error goes away